stable/kapacitor/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 name: kapacitor
-version: 1.2.1
+version: 1.2.2
 appVersion: 1.5.2
-description: InfluxDB's native data processing engine. It can process both stream
+deprecated: true
+description: DEPRECATED InfluxDB's native data processing engine. It can process both stream
```

Comment:
any new home to point to?

Reply:
Yep: https://github.com/helm/charts/tree/master/stable/kapacitor#deprecated---kapacitor

Already in the README from previous commit.